cffi/build.py

```diff
@@ -30,17 +31,16 @@ def search_paths(env_var: str) -> List[str]:
 EXTRA_CFLAGS += shlex.split(os.environ.get("LIBYANG_EXTRA_CFLAGS", ""))
 EXTRA_LDFLAGS = shlex.split(os.environ.get("LIBYANG_EXTRA_LDFLAGS", ""))
 
-with open(os.path.join(HERE, "source.c")) as f:
-    BUILDER.set_source(
-        "_libyang",
-        f.read(),
-        libraries=["yang"],
-        extra_compile_args=EXTRA_CFLAGS,
-        extra_link_args=EXTRA_LDFLAGS,
-        include_dirs=HEADERS,
-        library_dirs=LIBRARIES,
-        py_limited_api=False,
+with open(os.path.join(HERE, 'source.c')) as f:
+    ffibuilder.set_source('_libyang',
+                          f.read(),
+                          libraries=['yang'],
+                          extra_compile_args=EXTRA_CFLAGS,
+                          extra_link_args=EXTRA_LDFLAGS,
+                          include_dirs=HEADERS,
+                          library_dirs=LIBRARIES,
+                          py_limited_api=False,
```

Comment:
Please do not change lines unless required. Do not mix reformatting/refactoring with libyang2 related stuff. Also, I doubt this matches black coding style :)